jupyter-book/_toc.yml

```diff
@@ -56,6 +47,18 @@
         - file: python_scripts/cross_validation_sol_01
     - file: overfit/overfit_take_away
 
+- part: Fine tune your model
+  chapters:
+    - file: tuning/parameter_tuning_module_intro
+    - file: python_scripts/04_parameter_tuning
```

Comment:
We could rename this section `tuning/manual_parameter_tweaking` to contrast it with `tuning/automated_parameter_tuning` for the next notebook (and change their titles accordingly). WDYT?

Reply:
we can remove the `04_` prefix now.

Reply:
I would prefer to use `parameter_tuning_manual` to use `parameter_tuning_` as a prefix if possible